nengo_viz/static/viz_modal.js

```diff
@@ -61,6 +61,20 @@ VIZ.Modal.prototype.text_body = function(text, type) {
     $p.append(document.createTextNode(text));
 }
 
+VIZ.Modal.prototype.help_body = function() {
+    this.clear_body();
+
+    $('.modal-dialog').addClass('modal-sm');
```

Comment:
Cool thanks!
On Jun 3, 2015 10:28 AM, "Trevor Bekolay" <notifications@github.com> wrote:

> In nengo_viz/static/viz_modal.js
> <https://github.com/nengo/nengo_viz/pull/281#discussion_r31628072>:
>
> > @@ -61,6 +61,20 @@ VIZ.Modal.prototype.text_body = function(text, type) {
> >      $p.append(document.createTextNode(text));
> >  }
> >
> > +VIZ.Modal.prototype.help_body = function() {
> > +    this.clear_body();
> > +
> > +    $('.modal-dialog').addClass('modal-sm');
>
> $('.modal-dialog') selects every element on the page that has modal-dialog
> in the class list. If in the future we add multiple modals, then this is
> going to affect all of those modals. It can also be a bit slow to search
> the entire page if we have a lot of elements. By using find we're only
> searching for elements with the modal-dialog class that are a descendant
> of this.$div.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nengo/nengo_viz/pull/281/files#r31628072>.
>